docs/conventions.rst

```diff
@@ -217,12 +218,12 @@ XDG base directories
 standard locations for user-specific application data. Popular toolkits provide
 convenience functions for accessing XDG base directories. These include:
 
-- Electron: XDG base directories can be accessed with ``app.getPath``
```

Comment:
convenience -> convenient in the above line.

Reply:
`convenience function` is the correct technical term: [Wikipedia](http://en.wikipedia.org/wiki/Convenience_function).